src/locales/core.ts

```diff
@@ -1,11 +1,16 @@
 import { createI18n } from 'vue-i18n'
-import uaLocale from './ua.json'
+import uaLocale from '@/locales/ua.json'
+import { numbersFormats } from '@/locales/rules/numbersFormats'
+import { pluralizationRules } from '@/locales/rules/pluralizationRules'
 
+const defaultLocale = 'uk'
 export const i18n = createI18n({
   legacy: false,
-  locale: 'ua',
-  fallbackLocale: 'ua',
+  locale: defaultLocale,
+  fallbackLocale: defaultLocale,
   messages: {
-    ua: uaLocale
-  }
+    [defaultLocale]: uaLocale
+  },
+  numberFormats: numbersFormats,
```

Comment:
если назовем переменную numberFormats и файл тогда желательно переименовать так же, тогда сможем описать тут как: 
```suggestion
  numberFormat,
```